CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.7.6 (2020-10-14)
+* Update corresponding ingress when site URL is changed
+* Delete corresponding ingress when a site is deleted
+
 ## 0.7.5 (2020-09-29)
 * Update openstad logo
 
```

Comment:
@rudivanhierden  dank voor toevoegen! Als je naar Development doet mag de kop voor gemak ##unreleased zijn, versie nr komt pas als we naar master gaan en daadwerkelijk ook de tag toevoegen, die wordt dan ook automatisch gebuild en naar docker gepushed.